plugins/by-name/glow/default.nix

```diff
@@ -35,28 +37,31 @@ helpers.neovim-plugin.mkNeovimPlugin {
       "single"
     ] "Style of the floating window's border.";
 
-    style = helpers.defaultNullOpts.mkEnum [
-      "dark"
-      "light"
-    ] (helpers.mkRaw "vim.opt.background") "Glow style.";
+    style = defaultNullOpts.mkNullable (
+      with lib.types;
+      either (maybeRaw str) (enum [
+        "dark"
+        "light"
+      ])
+    ) (lib.nixvim.mkRaw "vim.opt.background") "Glow style.";
```

Comment:
Using str and an enum of strings together feels odd. Should the option just be using `mkStr`?

Reply:
I thought so too, but I didn't know if this helped show the default 2 options from the plugin better.